src/libs/actions/Policy.ts

```diff
@@ -2403,7 +2403,7 @@ function buildOptimisticPolicyRecentlyUsedTags(policyID?: string, transactionTag
         }
 
         const tagListKey = policyTagKeys[index];
-        newOptimisticPolicyRecentlyUsedTags[tagListKey] = [...new Set([...tag, ...(policyRecentlyUsedTags[tagListKey] ?? [])])];
+        newOptimisticPolicyRecentlyUsedTags[tagListKey] = [...new Set([tag, ...(policyRecentlyUsedTags[tagListKey] ?? [])])];
```

Comment:
Why this change?

Reply:
I got another bug and fixed. spreading tag means every letter in the tag will be included in the list that's why recently used tags now doesn't work BTW.